services/sidecar/src/simcore_service_sidecar/cli.py

```diff
@@ -41,16 +55,29 @@ async def run_sidecar(
         project_id,
         node_id,
     )
-    async with DBContextManager() as db_engine:
-        async with RabbitMQ(config=RABBIT_CONFIG) as rabbit_mq:
-            next_task_nodes: Optional[List[str]] = await inspect(
-                db_engine, rabbit_mq, job_id, user_id, project_id, node_id=node_id
-            )
-            log.info(
-                "COMPLETED task %s processing for user %s, project %s, node %s",
-                job_id,
-                user_id,
-                project_id,
-                node_id,
-            )
-            return next_task_nodes
+
+    abortion_task = (
+        asyncio.get_event_loop().create_task(
+            perdiodicaly_check_if_aborted(is_aborted_cb)
+        )
+        if is_aborted_cb
+        else None
+    )
+    try:
+        async with DBContextManager() as db_engine:
```

Comment:
NOTE: we should get rid of this DBContextManager asap!

Reply:
Will you be kind enough in creating an issue and explaining why it needs to be removed?

Reply:
@pcrespov : also don't really see why? it creates an engine, and then removes it once the work is done. why should I keep it up?
if that is what you mean.

Reply:
There should be one engine per application that acquires/releases connections from its pool when needed.
Creating/destroying engines for every single connection is not recommended since it is costly

Reply:
As a less intrusive change, I would propose to create the db_engine only the first time this context manager is acquired. For subsequent calls the first acquired instance is reused.
The db_engine will be stored as a class property.

Not that pretty but solves the issues.

Reply:
Now that I'm looking better at it. There is one implementation just like that in the `simcore-sdk`, the one in the `sidecar` does not have it.

Reply:
ok then I will fix this. I though the engine was ok.